src/gwt/test/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/ChunkContextUiTests.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * ChunkContextUiTests.java
+ *
+ * Copyright (C) 2009-20 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+package org.rstudio.studio.client.workbench.views.source.editors.text.rmd;
+
+import com.google.gwt.junit.client.GWTTestCase;
+
+
+public class ChunkContextUiTests extends GWTTestCase
+{
+   @Override
+   public String getModuleName()
+   {
+      return "org.rstudio.studio.RStudioTests";
+   }
+   
+   public void testExtractChunkLabel()
```

Comment:
Let's add a couple more here to increase code coverage -- maybe one in which there's no label, and another in which there are no options?

```
{r echo=TRUE}
{r thisisalabel}
```

Reply:
Updated